### PR TITLE
Appconfig store

### PR DESCRIPTION
**Pull request by abhi-intelops**

### CODE REVIEW

File: capten/agent/pkg/capten-store/app_config_store.go [hunk 1]
```diff
@@ -130,13 +130,16 @@ func formUpdateKvPairs(config *agentpb.SyncAppData) (string, bool) {
...
 	}
 
-	if config.Config.CreateNamespace {
-		params = append(params,
-			fmt.Sprintf("%s = 'true'", createNamespace))
-	}
-	if config.Config.PrivilegedNamespace {
-		params = append(params,
-			fmt.Sprintf("%s = 'true'", privilegedNamespace))
+	{
```
Comment (vramk23) [inline capten/agent/pkg/capten-store/app_config_store.go:133]: remove this additional brackets

**Review by vramk23 — CHANGES_REQUESTED** [review]

File: capten/agent/pkg/capten-store/app_config_store.go [hunk 2]
```diff
@@ -130,13 +130,16 @@ func formUpdateKvPairs(config *agentpb.SyncAppData) (string, bool) {
...
+
+		if config.Config.CreateNamespace {
+			params = append(params,
+				fmt.Sprintf("%s = true", createNamespace))
+		}
+		if config.Config.PrivilegedNamespace {
+			params = append(params,
+				fmt.Sprintf("%s = true", privilegedNamespace))
+		}
```
[the remark this answers is not in the document]
Comment (abhi-intelops) [reply capten/agent/pkg/capten-store/app_config_store.go:142]: it's adding that value only when it is true, if not then it is false by default, thanks to "zero" values in golang

File: capten/agent/pkg/capten-store/app_config_store.go (hunk 1 above)
Comment (abhi-intelops) [reply capten/agent/pkg/capten-store/app_config_store.go:133]: removed

File: capten/agent/pkg/capten-store/app_config_store.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (abhi-intelops) [reply capten/agent/pkg/capten-store/app_config_store.go]: it will go inside if only if `PrivilegedNamespace` is true

**Pull request merged**